packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionTncSection/JurisdictionTncSection.tsx

```diff
@@ -29,7 +29,7 @@ const JurisdictionTncSection: FC<TProps> = ({ isCheckBoxChecked, selectedJurisdi
     const selectedCompany = companyNamesAndUrls[selectedJurisdiction as keyof typeof companyNamesAndUrls];
 
     return (
-        <div className='sticky flex flex-col justify-center mt-1500 gap-600 lg:h-2500 bottom-50 bg-system-light-primary-background pt-[15px] px-1000 pb-500 w-screen'>
+        <div className='flex flex-col justify-center mt-1500 gap-600 sticky bottom-50 bg-system-light-primary-background px-1000 pt-[15px] pb-500 w-screen sm:w-auto sm:bg-inherit sm:static sm:p-50 lg:h-2500'>
```

Comment:
The classes here can be quite confusing and abit bloated in my opinion. I suggest to remove `flex flex-col justify-center gap-600...`
and instead replace it with `text-center space-y-600`. HTML are already stacking ontop of each other by default unless the it is wrap with a `flex-row`. Writing less styles is always a good approach especially when you are using tailwind.

Reply:
Updated here, but there are some places where we need gap because space was conflicting with margin styles.